docs/en_US/rules/sinks/plugin/influx_v2.md

```diff
@@ -0,0 +1,112 @@
+# InfluxDB Sink
+
+The sink will publish the result into a InfluxDB.
+
+## Compile & deploy plugin
+
+Please make following update before compile the plugin,
```

Comment:
@superrxan Please review the compile information here and help to add the CI to build this plugin

Reply:
@superrxan Thank you for your reply. I will add information about the plug-in in makefile later :)

Reply:
@ngjaying After discussion, adding CI requires extensions Mod modification will be added in subsequent pr